lms/templates/instructor/instructor_dashboard_2/data_download.html

```diff
@@ -1,9 +1,33 @@
+<%namespace name='static' file='/static_content.html'/>
 <%page args="section_data" expression_filter="h"/>
 <%!
 from django.utils.translation import ugettext as _
 from openedx.core.djangolib.markup import HTML, Text
+from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
 %>
 
+% if configuration_helpers.get_value('CUSTOM_THEME_DISPLAY_SENSITIVE_DATA_MSG_FOR_DASHBOARD', settings.FEATURES.get('CUSTOM_THEME_DISPLAY_SENSITIVE_DATA_MSG_FOR_DASHBOARD', True)):
+
+<script type="text/javascript" src="${static.url('js/vendor/jquery.leanModal.js')}"></script>
+
+<style>
+.dataDisclaimer-modal{
+  color:#555;
+  padding: 60px;
+  line-height: 1.9;
+}
+</style>
```

Comment:
It's ok to put CSS here or we need it compiled?

Reply:
We need to put it in the static sass/css pipeline